torchgeo/datasets/nlcd.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""NLCD dataset."""
+
+import os
+from typing import Any, Callable, Dict, List, Optional
```

Comment:
Can you run:
```console
$ pyupgrade --py39-plus torchgeo/datasets/nlcd.py
```
You'll also need to remove the unused imports:
```suggestion
from typing import Any, Callable, Optional
```
Yay for Python 3.9!